torch/__future__.py

```diff
@@ -0,0 +1,15 @@
+"""
+This global flag controls whether to change the existing parameters
+in-place instead of assigning new tensors to the parameters when
+converting a module using `module._apply(fn)`.
+
+Default: False
+"""
+TORCH_FLAG_overwrite_module_params_on_conversion_ = False
+
+def set_overwrite_module_params_on_conversion(value):
```

Comment:
also, can you document this?  (This can be in a future PR).

Reply:
Yes I will document it.